public_dropin_environments/triton_server/README.md

```diff
@@ -0,0 +1,44 @@
+# NVIDIA Triton Inference Server Drop-In Template Environment
+
+This template environment can be used to create artifact-only custom models with 
+the following Triton Inference Server backends: ONNX, vLLM, TensorRT-LLM.
```

Comment:
This env doesn't support TensorRT-LLM or vLLM right? it's just the standard PY3 env from your base image. This is what upstream says, I think we should include similar language
> The xx.yy-py3 image contains the Triton inference server with support for Tensorflow, PyTorch, TensorRT, ONNX and OpenVINO models.

Reply:
updated